Yoast/wordpress-seo
# Add inline-link (nofollow / sponsored) block editor integration.

## Context


* Adds the possibility to add rel keywords `nofollow` and `sponsored` to individual links in the block editor.

![Screenshot 2020-06-02 at 20 22 14](https://user-images.githubusercontent.com/1488816/83555592-c61aed00-a50e-11ea-8ffe-742863330a95.png)



## Summary


This PR can be summarized in the following changelog entry:

* Adds the possibility to add rel keywords `nofollow` and `sponsored` to individual links in the block editor.

## Relevant technical choices:

* I simply copied another implementation of this and included it in our existing block editor integration.
* The current solution is definitely not desired, but since the WordPress block editor doesn't offer any other viable integration points, there wasn't much else to do than use formats API for this.
* I fixed a lot of codestyle. That is also why some unrelated code is touched.
* I moved all constant assignments from wp to regular imports based on externals.

## Test instructions

This PR can be tested by following these steps:

* Add a link.
* Toggle nofollow or sponsored on.
* See the toggle reflected on the frontend.

## UI changes

## Documentation

## Quality assurance


Fixes #
